### PR TITLE
Spanish site

### PR DESCRIPTION
**Pull request by fm**
Starting a PR for translated spanish site. Most of the pages are translated. Just missing the sample letters and a few minor updates which I'll do tomorrow. 

@baltpeter `solicituddedatos.es` is still available if you want to secure it while I wrap up the translations

### CODE REVIEW

**Comment by fm** [comment]
Just realized the JSON I downloaded Weblate may not be in the right format... how can we pull the Weblate translations into the correct JSON format for `es.json`?

**Comment by baltpeter** [comment]
Thank you very much, @fm!

> @baltpeter solicituddedatos.es is still available if you want to secure it while I wrap up the translations

Great, we have now bought that domain. Doesn't currently point anywhere but we have secured it.

> Just realized the JSON I downloaded Weblate may not be in the right format... how can we pull the Weblate translations into the correct JSON format for es.json?

You actually don't need to do that manually, Weblate will send a PR with the changes. That should happen automatically after a day but I have now manually requested a PR and merged the updates in #542. You should be able to just revert the changes to that file on your branch and then rebase on our master to get the updated file.

**Review by baltpeter — CHANGES_REQUESTED** [review]
Huge thanks once again, @fm—this is great! I have left some comments below. Also, some general notes:

* If you want, feel free to list yourself as an author for the blog posts you have translated. In the future, we will want to have a separate `translators` key but for now, we are simply handling this via the `authors`.
* For the other language versions, we have translated the slugs to make for nicer URLs. This works by setting `slug: "translated-slug"` and then additionally setting `aliases: ["original-english-slug"]` (plus any other aliases). It's fine if you don't want to do that though, as changing that retroactively is probably quite a bit of work.
* I think we should `last_edited` to the current time for the translated posts.
* The PDF templates should be fillable forms. But that is not blocking. I'll put that on my TODO list and submit a separate PR later, after this PR has been merged.
* The `.txt` templates should be human readable, see [here](https://www.datarequests.org/downloads/sample-letter-gdpr-erasure-request-datarequests.org.txt) for example.

File: src/i18n/en.json [hunk 1]
```diff
@@ -132,10 +132,10 @@
         "verein-home-card-mission-button": "Read more about our goals",
         "verein-home-card-donate-heading": "Donate",
         "verein-home-card-donate-content": "We want to keep our project free and open to everyone. If you want to help us do that and launch new projects, consider donating to us. We are very grateful for every donation.",
-        "verein-home-card-donate-button": "I want to donate.",
+        "verein-home-card-donate-button": "I want to donate",
         "verein-home-card-join-heading": "Become a member",
         "verein-home-card-join-content": "Our project is meant for participation. Whether you want to contribute and help with the development or you just want to be closer to the action, we will be happy to welcome you with us.",
-        "verein-home-card-join-button": "I want to join.",
+        "verein-home-card-join-button": "I want to join",
```
[the remark this answers is not in the document]
Comment (fm) [reply src/i18n/en.json:138]: You don't need the dots, hence why I removed them.

File: src/i18n/en.json [hunk 2]
```diff
@@ -132,10 +132,10 @@
         "verein-home-card-mission-button": "Read more about our goals",
         "verein-home-card-donate-heading": "Donate",
         "verein-home-card-donate-content": "We want to keep our project free and open to everyone. If you want to help us do that and launch new projects, consider donating to us. We are very grateful for every donation.",
-        "verein-home-card-donate-button": "I want to donate.",
+        "verein-home-card-donate-button": "I want to donate",
```
[the remark this answers is not in the document]
Comment (fm) [reply src/i18n/en.json:135]: You don't need the dots, hence why I removed them.

File: src/i18n/en.json [hunk 3]
```diff
@@ -39,7 +39,7 @@
         "fr": "French (Français)",
         "home-hero-line-1": "You have a right to",
         "home-hero-word": "data access",
-        "home-hero-line-2": "Use it.",
+        "home-hero-line-2": "Use it",
```
[the remark this answers is not in the document]
Comment (fm) [reply src/i18n/en.json:42]: You don't need the dots, hence why I removed them.

[the remark this answers is not in the document]
Comment (baltpeter) [reply src/i18n/en.json:42]: I know we don't _need_ them but we prefer to have them. :)

File: src/i18n/en.json (hunk 2 above)
[the remark this answers is not in the document]
Comment (fm) [reply src/i18n/en.json:135]: From a UI/UX perspective, the dots look out of place, two with, one without. And adding a dot to the missing entry would not work either. 

Screen Shot 2021-01-27 at 12 37 59 PM

**Comment by baltpeter** [comment]
Alright, I'll take over now and fix the merge conflicts, update the URLs in the footer and then prepare this for deploying.

**Comment by baltpeter** [comment]
I have now created the first deploy of the Spanish site. We'll have to wait for the DNS to update to see if everything worked correctly (and I'll need to manually request an SSL certificate once the DNS is updated).

**Pull request merged**